pallets/midnight/src/lib.rs

```diff
@@ -101,10 +98,9 @@ pub mod pallet {
 	#[cfg(hardfork_test)]
 	const STORAGE_VERSION: StorageVersion = StorageVersion::new(100);
 
-	pub const FIXED_MN_TRANSACTION_WEIGHT: Weight =
-		Weight::from_parts(WEIGHT_REF_TIME_PER_SECOND / 1000, 0);
+	pub const FIXED_MN_TRANSACTION_WEIGHT: Weight = Weight::from_parts(100_000_000_000, 0);
```

Comment:
It would be nice to add a comment calculating how many txs per block would fit in a block considering that Weight. For that value, it would be only 20 tx per block, which I think it is too conservative?